main.py

```diff
@@ -110,9 +130,28 @@ def run(self):
 def main():
   socket.setdefaulttimeout(None)
 
+  print 'Creating db connector'
+  db_connector = create_db_connector()
```

Comment:
Aren't they all going into the same DB that hosts our episodes and series?

Reply:
this one is the regular db and the other one is the podcast db